docs(k8s): 1.23 is EOS (#2913)
scaleway/docs-content: containers/kubernetes/reference-content/version-support-policy.mdx

  * Control planes cannot be upgraded on obsolete CNIs.
  * *Recommended action:* Create new clusters using a supported CNI, either via API or from the Scaleway console.

<Message type="important">

> Clusters still running unsupported CNIs (Flannel, Weave) do not have the ability to upgrade manually. While migrating to v1.25 (May 15th, 2024), Scaleway will change the [CNI configuration](https://www.scaleway.com/en/developers/api/kubernetes/#path-clusters-update-a-cluster-response-cni) to `none`. 

Reviewer: ```suggestion
Clusters still running unsupported CNIs (Flannel, Weave) cannot be upgraded manually. While migrating to v1.25 (May 15th, 2024), Scaleway will change the [CNI configuration](https://www.scaleway.com/en/developers/api/kubernetes/#path-clusters-update-a-cluster-response-cni) to `none`. 
```
